frontend/src/main/components/OurTable.js

```diff
@@ -42,7 +42,9 @@ export default function OurTable({ columns, data, testid = "testid", ...rest })
                     <tr {...headerGroup.getHeaderGroupProps()}>
                         {headerGroup.headers.map(column => (
                             <th
-                                {...column.getHeaderProps(column.getSortByToggleProps())}
+                                {...column.getHeaderProps({
+                                    ...column.getSortByToggleProps() // Include sorting functionality
```

Comment:
Changing this back to its original state might speed-up your mutation testing (since it theoretically won't run it for `OurTable.js` anymore)